pulsar-broker/src/main/java/org/apache/pulsar/broker/web/PulsarWebResource.java

```diff
@@ -791,6 +791,11 @@ protected void validateGlobalNamespaceOwnership(NamespaceName namespace) {
 
                     log.warn(msg);
                     validationFuture.completeExceptionally(new RestException(Status.PRECONDITION_FAILED, msg));
+                } else if (policies.deleted) {
+                    String msg = String.format("Namespace %s is deleted", namespace.toString());
+                    log.warn(msg);
+                    validationFuture.completeExceptionally(new RestException(Status.PRECONDITION_FAILED,
+                            "Namespace is deleted"));
```

Comment:
```suggestion
                            msg));
```

Reply:
Ah, this is the same with line- 805. That means no detailed namespace info in the response status text.